pkg/config/config.go

```diff
@@ -1101,21 +1101,8 @@ func Reload() (*Config, error) {
 }
 
 func (c *Config) ActiveDestination() (uri, identity string, machine bool, err error) {
```

Comment:
I think it's a good time to add a rather verbose comment to the function explaining what is being returned and in which order with respect to the environment variables.